packages/dx-react-grid-bootstrap3/src/templates/column-chooser-panel.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const ColumnChooserPanel = ({ columnChooserItems, onColumnToggle, itemTemplate }) => (
+  <div className="list-group">
+    {columnChooserItems.map(({ column, hidden }) => React.cloneElement(
+      itemTemplate({ column, hidden, onClick: onColumnToggle }),
```

Comment:
Why do you map `onColumnToggle` to `onClick` without binding arguments? It also seems that `onClick` should be renamed to `onVisibilityToggle: (toggle?: boolean) => void` and `onColumnToggle` may be named as `onColumnVisibilityToggle: (columnName, toggle?: boolean) => void`.